bpf/lib/ipv4.h

```diff
@@ -48,7 +63,7 @@ static __always_inline int ipv4_dec_ttl(struct __ctx_buff *ctx, int off,
 
 	new_ttl = ttl - 1;
 	/* l3_csum_replace() takes at min 2 bytes, zero extended. */
```

Comment:
Nit: This comment doesn't make sense anymore with the new function.

Reply:
Yeah I was unsure what to do with this, so left it for now. Maybe turn it into proper documentation for `ipv4_csum_update_by_value()`.